spikingjelly/activation_based/triton_kernel/flexsn/wrapper.py

```
@@ -1,24 +1,21 @@
-import torch
 from typing import Optional, Tuple
 
-from torch import autograd
+import torch
 
 try:
     import triton
 except BaseException as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `BaseException`

(BLE001)